invokeai/app/services/model_install/model_install_default.py

```diff
@@ -233,6 +233,29 @@ def import_model(self, source: ModelSource, config: Optional[Dict[str, Any]] = N
         self._install_jobs.append(install_job)
         return install_job
 
+    def get_hugging_face_models(self, source: str) -> List[AnyHttpUrl]:
```

Comment:
I think this could just live in the route function or in a util script. Just to avoid having to implement this abstract function if we ever make more/different install services.

Reply:
Yeah, shouldn't be on the service. It could be on `HuggingFaceMetadata`, though.

Reply:
Yeah that makes sense